jetbrains-core/tst/software/aws/toolkits/jetbrains/utils/rules/CloudFormationLazyInitRule.kt

```diff
@@ -53,10 +53,7 @@ class CloudFormationLazyInitRule(
 
         // wait for changeset creation to complete
         try {
-            cloudformationClient.waiter().waitUntilChangeSetCreateComplete {
-                it.stackName(stackName)
-                it.changeSetName(changeSetArn)
-            }.unwrapResponse()
+            cloudformationClient.waitForChangeSetCreateComplete(stackName, changeSetArn)
```

Comment:
comment with linkback to SDK issue on why?

Reply:
Added